MyGUIEngine/include/MyGUI_Widget.h

```diff
@@ -357,7 +358,7 @@ namespace MyGUI
 		Widget* _getClientWidget();
 		const Widget* _getClientWidget() const;
 
-		virtual void setPropertyOverride(const std::string& _key, const std::string& _value);
+		virtual void setPropertyOverride(std::string_view _key, std::string_view _value);
```

Comment:
A `virtual` means that the client applications may derive from class and override a virtual method. When you change the signature of virtual method, client applications which override it will stop to compile.

So when you modify a library's public API, you usually keep both old and new API, but mark old API as deprecated. Later you remove deprecated API as a part of cleanup for major release and post a migration guide. In this case developers of client applications have enough time to migrate their code to upstream API. For example, API from Qt4 was deprecated in Qt5 and removed only in Qt6.

So far MyGUI 3.4.3 is incompatible with MyGUI 3.4.2 and earlier versions, but release changelog does not contain any info about it.